package.json

```diff
@@ -17,12 +17,13 @@
   },
   "dependencies": {
     "async": "0.2.x",
-    "knex": "0.9.0",
-    "lodash": "3.10.1",
+    "bluebird": "3.7.1",
+    "knex": "0.21.1",
+    "lodash": "4.17.15",
     "mssql": "2.x.x",
```

Comment:
I wonder if we should move these other db driver deps to peerDeps or optionalDeps? We should probably also version pin them in the same way as the platform at least as this can cause version diffs.

Reply:
As discussed, we will be ensuring these deps match their respective platform versions